### PR TITLE
fix(router): fix lazy loading triggered by redirects from wildcard ro…

### PR DESCRIPTION
**Pull request by vsavkin**

### CODE REVIEW

File: modules/@angular/router/test/apply_redirects.spec.ts [hunk 1]
```diff
@@ -305,6 +305,34 @@ describe('applyRedirects', () => {
         expect((<any>config[0])._loadedConfig).toBe(loadedConfig);
       });
     });
+
+    it('should load the configuration after a local redirect from a whilecard route', () => {
```
Comment (vicb) [inline modules/@angular/router/test/apply_redirects.spec.ts]: type whilecard

File: modules/@angular/router/test/apply_redirects.spec.ts [hunk 2]
```diff
@@ -305,6 +305,34 @@ describe('applyRedirects', () => {
...
+
+      const config =
+          [{path: 'not-found', loadChildren: 'children'}, {path: '**', redirectTo: 'not-found'}];
+
+      applyRedirects(<any>'providedInjector', <any>loader, tree('xyz'), config).forEach(r => {
+        expect((<any>config[0])._loadedConfig).toBe(loadedConfig);
+      });
+    });
+
+    it('should load the configuration after an absolute redirect from a whilecard route', () => {
```
Comment (vicb) [inline modules/@angular/router/test/apply_redirects.spec.ts]: same typo

File: modules/@angular/router/test/integration.spec.ts [hunk 3]
```diff
@@ -1704,6 +1705,33 @@ describe('Integration', () => {
...
+             class LoadedModule {
+             }
+
+             loader.stubbedModules = {lazy: LoadedModule};
+             const fixture = createRoot(router, RootCmp);
+
+             router.resetConfig(
+                 [{path: 'lazy', loadChildren: 'lazy'}, {path: '**', redirectTo: 'lazy'}]);
+
+             router.navigateByUrl('/lazy/loaded');
```
Comment (vicb) [inline modules/@angular/router/test/integration.spec.ts]: is this path expected here ?

**Comment by vicb** [comment]
please add router & review labels for the PRs

File: modules/@angular/router/test/integration.spec.ts (hunk 3 above)
Comment (vsavkin) [inline modules/@angular/router/test/integration.spec.ts]: Yup

**Comment by alxhub** [comment]
Merged as 0f21a58.

**Pull request closed without merging**